ballerina/tests/55_declarative_authentication_pb.bal

```diff
@@ -0,0 +1,300 @@
+        var response = check self.sClient->receive();
+        if response is () {
+            return response;
+        } else {
+            [anydata, map<string|string[]>] [payload, headers] = response;
+            return payload.toString();
+        }
+    }
+
+    isolated remote function receiveContextString() returns ContextString|Error? {
+        var response = check self.sClient->receive();
+        if response is () {
+            return response;
+        } else {
+            [anydata, map<string|string[]>] [payload, headers] = response;
+            return {content: payload.toString(), headers: headers};
+        }
+    }
+
+    isolated remote function sendError(Error response) returns Error? {
+        return self.sClient->sendError(response);
+    }
+
+    isolated remote function complete() returns Error? {
+        return self.sClient->complete();
+    }
+}
+
+public client class HelloWorld55StringCaller {
+    private Caller caller;
+
+    public isolated function init(Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendString(string response) returns Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextString(ContextString response) returns Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(Error response) returns Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public client class HelloWorld55EmptyScopeStringCaller {
+    private Caller caller;
+
+    public isolated function init(Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendString(string response) returns Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextString(ContextString response) returns Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(Error response) returns Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+// public type ContextStringStream record {|
+//     stream<string, error?> content;
+//     map<string|string[]> headers;
+// |};
+
+// public type ContextString record {|
+//     string content;
+//     map<string|string[]> headers;
+// |};
```

Comment:
Shall we remove this commented code-segment ?

Reply:
These are there in all the stub files since they are generated automatically.